tests/MavenTests/testLoadSamples.cpp

```diff
@@ -121,13 +121,15 @@ void TestLoadSamples:: testMinMaxRT() {
 
 }
 
+#ifndef WIN32
```

Comment:
@rish9511 Should fix the test on Windows, no?

Reply:
Yes, separate PR. Could you create an issue for it or put it as part of our requirements somewhere so this is taken care of in the next release?

Reply:
Created an [issue](https://github.com/ElucidataInc/ElMaven/issues/905) for this . Will try 
to fix this before the next release